src/routes/search/_RangeFilter.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
```

Comment:
Same - add some context when range filter should be used, what are the inputs needed, etc.